Documentation/Books/Manual/Administration/Encryption/README.md

```diff
@@ -12,15 +12,21 @@ but when the data is already on disk, you also need protection.
 That is where the Encryption feature comes in. 
 
 The Encryption feature of ArangoDB will encrypt all data that ArangoDB 
-is storing in your database before it is written to disk.
+is storing in your database before it is written to disk. This feature applies to all ArangoDB Deployments.
```

Comment:
applies to all ArangoDB **deployment modes**?

Reply:
Yes, because encryption at rest is a feature of `arangod`. This has been confirmed from Ewout

Reply:
Yeah, it is a arangod EE feature. My actual question was whether it should be called "deployments" or "deployment modes". We call single server / active failover / dc2dc / cluster the ArangoDB deployment modes elsewhere in the docs.

Reply:
ah ok, I thought it was a question, my fault. 

Yes calling it **deployment modes** sounds good to me.